src/commands/admin/prune.ts

```diff
@@ -0,0 +1,107 @@
+import { Command } from '@lib/types/Command';
+import { ROLES } from '@root/config';
+import { adminPerms } from '@root/src/lib/permissions';
+import { GuildMember, Message, MessageActionRow, MessageButton, MessageEmbed } from 'discord.js';
+
+const PRUNE_TIMEOUT = 30;
+
+export default class extends Command {
+
+	description = `Prunes all members who don't have the <@&${ROLES.VERIFIED}> role`;
+	runInDM = false;
+
+	permissions(msg: Message): boolean {
+		return adminPerms(msg);
+	}
+
+	async run(msg: Message): Promise<void> {
+		let timeout = PRUNE_TIMEOUT;
+		await msg.guild.members.fetch();
+		const toKick = msg.guild.members.cache.filter(member => !member.user.bot && !member.roles.cache.has(ROLES.VERIFIED));
+
+		const confirmEmbed = new MessageEmbed()
+			.setTitle(`Server prune will kick ${toKick.size} members from the guild. Proceed?`)
+			.setColor('RED')
+			.setFooter(`This command will expire in ${PRUNE_TIMEOUT} seconds`);
```

Comment:
Might be worth making consistent with line 102? Since this starts off as "This command will expire in 30 seconds", not "30s"

Reply:
Give me a second :)